lib/tasks/addon-install.js

```diff
@@ -34,22 +34,23 @@ module.exports = Task.extend({
 
     ui.startProgress(chalk.green('Installing addon package'), chalk.green('.'));
 
+    var willSave = this.project.isEmberCLIAddon() && commandOptions.save;
     return npmInstall.run({
       packages: packageNames,
-      'save-dev': true,
+      'save-dev': !willSave,
       'save-exact': true
     }).then(function() {
       return this.project.reloadAddons();
     }.bind(this)).then(function() {
-      return this.installBlueprint(blueprintInstall, packageNames, blueprintOptions);
+      return this.installBlueprint(blueprintInstall, packageNames);
```

Comment:
I believe the blueprintOptions were needed in situations where you want the [raw command info](https://github.com/npm/nopt#the-rest-of-the-args) to pass along to the install task. It seems the reason it didn't break is because we don't have any coverage for that specific case (where the addon's default blueprint needs the raw addon command options). You'll need to pass in the command options in place of the blueprint options for it to pick up on the raw options passed in the addon installation.